tests/queries/0_stateless/03568_udf_memory_tracking.reference

```diff
@@ -0,0 +1,5 @@
+Row 1:
+──────
+queries:            0
```

Comment:
Why queries is 0?

Reply:
Maybe because `log_comment` should be `03568_udf_memory_tracking.sql`?
But I think that this condition can be simply removed and replaced with `kind = 'Select'`

Reply:
Ye, something went terribly wrong with this test, both test and reference are updated.